lynx/build.gradle

```diff
@@ -3,10 +3,10 @@ apply plugin: 'checkstyle'
 
 android {
   compileSdkVersion 24
-  buildToolsVersion "24.0.0"
+  buildToolsVersion "24.0.1"
 
   defaultConfig {
-    minSdkVersion 8
+    minSdkVersion 10
```

Comment:
Yes, it was a forced one:

https://github.com/pedrovgs/Renderers/blob/master/renderers/build.gradle